pandas/core/frame.py

```diff
@@ -5054,7 +5054,9 @@ def infer(x):
     def append(self, other, ignore_index=False, verify_integrity=False):
         """
         Append rows of `other` to the end of this frame, returning a new
-        object. Columns not in this frame are added as new columns.
+        object.
+
+        Columns not in this frame are added as new columns.
```

Comment:
Instead of "this frame" can you say "the caller"? Also while conceptually simple the usage of the word "Columns" mixes reference to the calling object and the object being appended, so differentiating between those would be preferable